package.json

```diff
@@ -12,11 +12,12 @@
     "styles:dev": "postcss example/assets/styles/style.css -c -o example/site.min.css -w",
     "styles:build": "NODE_ENV=production postcss example/assets/styles/style.css -c -o example/site.min.css",
     "scripts:dev": "rollup -c -w",
-    "scripts:build": "NODE_ENV=production rollup -c && minify dist/micromodal.js --output dist/micromodal.min.js",
+    "scripts:build:es": "NODE_ENV=production TARGET=es rollup -c",
+    "scripts:build:umd": "NODE_ENV=production TARGET=umd rollup -c && minify dist/micromodal.js --output dist/micromodal.min.js",
     "demo:scripts": "minify dist/micromodal.js example/assets/scripts/prism.js --output example/site.min.js",
     "demo:watch": "watch 'yarn demo:scripts' dist/",
     "livereload": "reload -b -s --dir example --port 5000",
-    "build": "concurrently \"yarn scripts:build\" \"yarn styles:build\"",
+    "build": "concurrently \"run-p scripts:build:*\" \"yarn styles:build\"",
```

Comment:
is `run-p` a thing? or should this be `yarn`?

Reply:
`run-p` comes from '`npm-run-all``run-p` comes from '`npm-run-all`. Runs all the tasks in parallel which matches the regex.